### ISSUE TITLE
Deprecation Symfony 6.2 - AsyncPersistPageHandler

### ISSUE THREAD

**Issue by youlead-bow** [FriendsOfSymfony/FOSElasticaBundle#1922]
In my project in Symfony 6, I have had this deprecation for some time:

> The "FOS\ElasticaBundle\Message\Handler\AsyncPersistPageHandler" class implements "Symfony\Component\Messenger\Handler\MessageHandlerInterface" that is deprecated since Symfony 6.2, use the {@see AsMessageHandler} attribute instead.

**Comment by boskos-q** [comment]
Just use attribute `#[AsMessageHandler]` and remove `implements MessageHandlerInterface`

**Comment by youlead-bow** [comment]
Yes the deprecation comes from the bundle, you simply have to replace the line in class AsyncPersistPageHandler:
`class AsyncPersistPageHandler implements MessageHandlerInterface`
by:
```
#[AsMessageHandler]
class AsyncPersistPageHandler
```

**Comment by XWB** [comment]
Was fixed in https://github.com/FriendsOfSymfony/FOSElasticaBundle/commit/4164ca0327808c59a8b331f3c91813ac2101299e
